### DISCUSSION TITLE
Q4 Community Manager Proposal

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#41] [General]
# 1. Objectives & Actions
## 1.1 DCDAO Development
Key Action 
-Promote the DCDAO Advocate Program, especially in Southeast Asia, and Latin America to grow DCDAO team members.
-Co-host AMA or Twitter space with other organizations and gain more inspiration from the other DAO
-Update the latest news weekly and share more interesting and helpful information with users.
-Launch campaign in Galex regularly.
## 1.2 Community Education
Key Action
-Let the new members learn what Darwinia is and DCDAO's vision in an easier way. (maybe like a quiz game in Discord)
-Increase the Interaction with members and collect their common questions. 
## 1.3 Ecosystem Development 
Key Action
-Collaborate with potential partners and expand Darwinia’s influence.
Polkadex / https://invarch.network/ 
DEX/Bridger/GameFi
# 2. Funding Support
The funding support is approximately 5000 USD/month. Here are the details.
## 2.1 Ambassador Program Monthly Expense
Within 2000USDT-3000USDT/month(the rewards will be paid by Ring)
## 2.2 Community Manager Monthly Salary
1700 USD
My meta mask wallet address： 0x7C548fe9aEd9F6C50cD372196eA5510ac56dBa8F（eth)
## 2.3 Other Expenses**
In order to activate our community and promote DCDAO, I plan to do some events like giveaways or other types of interaction with members in Q4. 
Budget: 180,000 Ring for Q4
DCDAO Marketing Multi-sig Address:
ETH:0x7AC72B45E72dAa1CA2a26FB543E7B096570d1BCd
Polygon:0xbaCd8aBCf84a1ED6B4daB676489f420163e3625b

**Comment by SasoLithops** [comment]
Hi!

I agree with proposed, except I would cross of Galxe since it was compromised a while ago and not sure if people would trust to connect their wallets again.

**Reply by Cindy884** [reply]
I see. Thks for your feedback, I have created DCDAO Multi-sig wallet and I think it will protect DCDAO's marketing budget from scam.

**Comment by samestshares** [comment]
Hi,
Galxe won't be the right platform to use this time following what happened, I will suggest using Zealy or Dework,  this will bring more people and get darwinia information closer to them

**Reply by Cindy884** [reply]
Thks for your suggestion~
